.circleci/config.yml

```diff
@@ -175,7 +175,9 @@ jobs:
                 echo "Skipping - running only in container number 0"
                 exit 0
             fi
+            python copy_beta.py $CIRCLE_ARTIFACTS
```

Comment:
why is that a python script? just do it in bash
also, move this to release notes step

Reply:
its there until the `create_artifats` in demisto-sdk will be merged and released.